.github/workflows/coveralls.yml

```diff
@@ -15,6 +14,8 @@ jobs:
         steps:
             - name: Checkout PR branch
               uses: actions/checkout@v4
+              with:
```

Comment:
We don't need to check out the entire history for what we're doing here.

Reply:
I think it works the other way around - setting `fetch-depth: 0` tells Git to fetch the full history (all commits, branches, and tags). The default is `1`, which only fetches the latest commit from the current branch.
(source: https://github.com/actions/checkout)